src/ORM/Queries/SQLExpression.php

```diff
@@ -84,7 +84,7 @@ abstract public function isEmpty();
      * Generate the SQL statement for this query.
      *
      * @param array $parameters Out variable for parameters required for this query
-     * @return string The completed SQL query
+     * @return string|null The completed SQL query or null if there's no DB connector or the query was empty
```

Comment:
The method returns null under these conditions already - I was confused why I was getting null so had to go look. This will save other developers the same fate.